tests/test_util.py

```diff
@@ -654,3 +658,15 @@ def test_label_formatter(labels, test_string, expected):
     else:
         with pytest.raises(KeyError):
             LabelFormatter().vformat(test_string, [], labels)
+
+
+@pytest.mark.parametrize(('retry', 'expected'), (
+    (5, 5),
+    (0, 0),
+    (1, 1),
+    (12, 12),
+))
+def test_retry_value(retry, expected):
```

Comment:
What does this test do? `expected` is not used anywhere

Reply:
I guess it is useless now, when we don't have any sanity check and not allowing strings, will delete that